examples/IBFE/explicit/ex5/example.cpp

```diff
@@ -678,8 +711,8 @@ postprocess_data(Pointer<Database> input_db,
     std::vector<const std::vector<libMesh::VectorValue<double> >*> grad_var_data;
 
     TensorValue<double> FF, FF_inv_trans;
-    boost::multi_array<double, 2> x_node, U_node;
-    VectorValue<double> F, N, U, n, x;
+    boost::multi_array<double, 2> x_node, X0_node, U_node, TAU_node;
```

Comment:
`X0` should just be `X`. `X` indicates reference coordinates --- there is no need to affix a `0`.